gateway/integration_test.go

```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/ory/dockertest/v3"
 	"github.com/stretchr/testify/require"
 
-	kitHelper "github.com/rudderlabs/rudder-go-kit/testhelper"
+	kit_helper "github.com/rudderlabs/rudder-go-kit/testhelper"
```

Comment:
Aren't we adhering to the outcome of the poll  :P

Reply:
enjoy!

Reply:
We voted for the same thing but lost :'(